flink-autoscaler/src/test/java/org/apache/flink/autoscaler/MetricsCollectionAndEvaluationTest.java

```diff
@@ -599,6 +602,73 @@ public void testMetricCollectionDuringStabilization() throws Exception {
         assertEquals(2, stateStore.getCollectedMetrics(context).size());
     }
 
+    @Test
+    public void testMetricCollectionDuringForbidden() throws Exception {
```

Comment:
I think this test should be part of the `JobAutoscalerImplTest` and then it can be simplified a lot.

Reply:
I want to test this config along with config STABILIZATION_INTERVAL and METRICS_WINDOW, so put it here